spec/lib/efile/nj/nj1040_calculator_spec.rb

```diff
@@ -1605,7 +1605,43 @@ def over_65_birth_year
       end
     end
 
-    context 'when no w2s' do
+    context 'when has only 1099-Rs' do
+      let(:intake) { create(:state_file_nj_intake, :df_data_minimal)}
+      let!(:state_file_1099r) { create :state_file1099_r, intake: intake, state_tax_withheld_amount: 100.1 }
+      let!(:state_specific_followup) { create :state_file_nj1099_r_followup, state_file1099_r: state_file_1099r, income_source: :military_pension }
```

Comment:
is it necessary to have the followup here? it seems like the method doesn't take this into account